modify default configuration
- do not grab images from desktop
- do not show error output of hacks
- ignore not installed hacks (they might disappear on update)
- set path to images to use
- display date + hostname in hacks requiring text input
- select default screensaver to use

================================================================================
Index: driver/XScreenSaver.ad.in
===================================================================
--- driver/XScreenSaver.ad.in.orig	2009-09-08 06:53:15.000000000 +0200
+++ driver/XScreenSaver.ad.in	2009-12-24 17:10:18.000000000 +0100
@@ -37,7 +37,7 @@
 *dpmsStandby:		2:00:00
 *dpmsSuspend:		2:00:00
 *dpmsOff:		4:00:00
-*grabDesktopImages:	True
+*grabDesktopImages: False
 *grabVideoFrames:	False
 *chooseRandomImages:	@DEFAULT_IMAGES_P@
 *imageDirectory:	@DEFAULT_IMAGE_DIRECTORY@
@@ -53,10 +53,10 @@
 *splash:		True
 *splashDuration:	0:00:05
 *visualID:		default
-*captureStderr: 	True
-*ignoreUninstalledPrograms: False
+*captureStderr: 	False
+*ignoreUninstalledPrograms: True
 
-*textMode:		file
+*textMode:		date
 *textLiteral:		XScreenSaver
 *textFile:		@DEFAULT_TEXT_FILE@
 *textProgram:		fortune
@@ -140,6 +140,8 @@ GetViewPortIsFullOfLies: False
 !
 *installColormap:	True
 
+*mode:                  one
+*selected:              150
 
 ! This is the list of installed screen saver modes.  See "man xscreensaver"
 ! for the syntax used here.
@@ -294,7 +296,7 @@ GetViewPortIsFullOfLies: False
 @GL_KLUDGE@ GL: 				gleidescope -root			    \n\
 @GL_KLUDGE@ GL: 				glknots -root				    \n\
 @GL_KLUDGE@ GL: 				glmatrix -root				    \n\
-- GL: 				glslideshow -root			    \n\
+- GL: 				glslideshow -root -duration 30 -zoom 60 -pan 30 -fade 5			    \n\ 
 @GL_KLUDGE@ GL: 				hypertorus -root			    \n\
 - GL: 				jigglypuff -root			    \n\
 				metaballs -root				    \n\
Index: driver/XScreenSaver_ad.h
===================================================================
--- driver/XScreenSaver_ad.h.orig	2009-09-08 06:54:40.000000000 +0200
+++ driver/XScreenSaver_ad.h	2009-12-24 17:11:15.000000000 +0100
@@ -1,7 +1,8 @@
 "#error Do not run app-defaults files through xrdb!",
 "#error That does not do what you might expect.",
 "#error Put this file in /usr/lib/X11/app-defaults/XScreenSaver instead.",
-"*mode:			random",
+"*mode:			one",
+"*selected:             150",
 "*timeout:		0:10:00",
 "*cycle:			0:10:00",
 "*lockTimeout:		0:00:00",
@@ -10,10 +11,10 @@
 "*dpmsStandby:		2:00:00",
 "*dpmsSuspend:		2:00:00",
 "*dpmsOff:		4:00:00",
-"*grabDesktopImages:	True",
+"*grabDesktopImages:	False",
 "*grabVideoFrames:	False",
 "*chooseRandomImages:	True",
-"*imageDirectory:	/Library/Desktop Pictures/",
+"*imageDirectory:	/usr/share/wallpapers/",
 "*nice:			10",
 "*memoryLimit:		0",
 "*lock:			False",
@@ -27,8 +28,8 @@
 "*splashDuration:	0:00:05",
 "*visualID:		default",
 "*captureStderr: 	True",
-"*ignoreUninstalledPrograms: False",
-"*textMode:		file",
+"*ignoreUninstalledPrograms: True",
+"*textMode:		date",
 "*textLiteral:		XScreenSaver",
 "*textFile:		",
 "*textProgram:		fortune",
@@ -193,7 +194,7 @@
   GL: 				gleidescope -root			    \\n\
   GL: 				glknots -root				    \\n\
   GL: 				glmatrix -root				    \\n\
-- GL: 				glslideshow -root			    \\n\
+- GL: 				glslideshow -root -root -duration 21 -zoom 84 -pan 30 -titles    \\n\
   GL: 				hypertorus -root			    \\n\
 - GL: 				jigglypuff -root			    \\n\
 				metaballs -root				    \\n\
